{{cookiecutter.folder_name}}/README.md

```diff
@@ -10,16 +10,16 @@
 ## Editing this Paper
 
 Fork and/or clone the project repo, and then
-edit the primary file. The name of this file will vary according to its format, but it should be one of either `index.rst` (if it's a reStructuredText Note), `index.md` (if it's a Markdown Note), `index.ipynb` (if it's a jupyter notebook Note) or `main.tex` (if it's a latex Note or paper).
+edit the primary file. The name of this file will vary according to its format, but it should be one of either `main.rst` (if it's a reStructuredText Note), `main.md` (if it's a Markdown Note), `main.ipynb` (if it's a jupyter notebook Note) or `main.tex` (if it's a latex Note or paper).
 
-For the non-latex formats, please use the `_static` folder for your images. For latex, you can go ahead and make sub-folders for figures as usual.
+Please use the `figures` folder for your images.
 
 ## Building this Paper
 
-GitHub is our primary publisher for LSST DESC Notes in rST, Md or ipynb formats: once the Note has been merged into the project repo's master branch, it will be visible as a "published" paper. This presentation may be improved later, as the LSST DESC Publication System evolves.
+GitHub is our primary distributor for LSST DESC Notes in `rST`, `Markdown` or `ipynb` formats: once the Note has been merged into the project repo's master branch, it will be visible as a *shared* (but not *published*) paper. The presentation of Notes will be improved later, as the LSST DESC Publication System evolves.
```

Comment:
should `rST` be `reStructuredText` if `Md` is `Markdown` (same for IPython Notebook? Maybe "[`reStructuredText`](http://docutils.sourceforge.net/rst.html) (`.rst`), [`Markdown`]() (`.md`), or an [`IPython Notebook`](https://ipython.org/notebook.html) (`.ipynb`)."